### PR TITLE
feat: Added Service Id Copy

### PR DESCRIPTION
**Pull request by OmriBarZik**
# Summary
Added the functionally to copy service id.

## Proposed Changes

  - Keyboard shortcut to copy service id

  - Renamed copy command from `copy container id` to `copy id`

 - Update help widget

## Checklist

- [ ] I added tests
- [ ] I updated the README if necessary
- [ ] This PR introduces a breaking change
- [X] Fixed issue #152 
- [ ] I added a picture of a cute animal cause it's fun

### CODE REVIEW

File: src/widgetsTemplates/help.widget.template.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ class myWidget extends baseWidget() {
     ▸ /:       Search the containers list view
     ▸ i:       Display information dialog about the selected container or service
     ▸ ⏎:       Show the logs of the current container or service
-    ▸ c:       Copy container id to the clipboard
+    ▸ c:       Copy id to the clipboard
```
[the remark this answers is not in the document]
Comment (OmriBarZik) [reply src/widgetsTemplates/help.widget.template.js:110]: I think it's too long and will be problematic if we want to add other views in the future

File: widgets/toolbar.widget.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ class myWidget extends baseWidget(EventEmitter) {
         keys: ['-'],
         callback: () => { this.emit('key', '-') }
       },
-      'copy container id': {
+      'copy id': {
```
[the remark this answers is not in the document]
Comment (OmriBarZik) [reply widgets/toolbar.widget.js:47]: I think Changing the wording on each view is the best solution.

[the remark this answers is not in the document]
Comment (lirantal) [reply widgets/toolbar.widget.js:47]: Yep. Planning to update the PR for that or would you want me to land this first, and then we can figure out that change?

[the remark this answers is not in the document]
Comment (OmriBarZik) [reply widgets/toolbar.widget.js:47]: I can do it in this PR but it may take me a while to do it

[the remark this answers is not in the document]
Comment (lirantal) [reply widgets/toolbar.widget.js:47]: Let's go ahead with merging then and we can follow-up with a PR later on

**Review by lirantal — APPROVED** [review]
LGTM

File: widgets/toolbar.widget.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (OmriBarZik) [reply widgets/toolbar.widget.js:47]: fixes in #168

**Pull request merged**